### ISSUE TITLE
token0price and token1price

### ISSUE THREAD

**Issue by veridelisi** [Uniswap/uniswap-v3-subgraph#39]
{
  pools(where:{id:"0x8ad599c3a0ff1de082011efddc58f1908eb6e6d8"}) {
    token0Price
    token1Price
    totalValueLockedToken1
    totalValueLockedToken0
    totalValueLockedETH
    token0{name}
    token1{name}
}
}

![Ekran Görüntüsü - 2021-08-01 00-14-32](https://user-images.githubusercontent.com/72710290/127753492-697027e9-3afc-4ccb-841c-5455db593a72.png)

I think token0price must be "0.0004" and token1price must be "2490"

**Comment by albydeca** [comment]
I agree. There are many more examples like this including the one below that makes no sense to me. What is the unit of measure of the prices?
